BranchMetrics/ios-branch-deep-linking-attribution
# SDK 2228 DMA Test automation

## Reference
SDK-2228 -- DMA Compliance testing automation
https://branch.atlassian.net/browse/SDK-2228

## Summary
- Added buttons in Test Bed App -
    * Set DMA Params - It calls Branch.setDMAParamsForEEA API
    * Send V2 Event - It calls BranchEvent.standardEvent API
- Updated logging Function and moved initialization code to Button click events.
- Changed linking method to cocoapods from static lib.
Note : Ref is added to branch 'SDK-2228-iOS-DMA-Compliance-Automation' in workflow for testing purpose only. It will be reverted back before merging into master branch.

## Type Of Change

## Testing Instructions
Run workflow from this branch manually and verify in logs( or Browserstack) that Set DMA Params test passed.










cc @BranchMetrics/saas-sdk-devs for visibility.
